gitpod-io/gitpod
# [supervisor-frontend] add user id for browser code

Co-authored-by: mustard <mhqnwt@gmail.com>
Co-authored-by: Jean Pierre <jeanp413@hotmail.com>

## Description


First step of [internal chat](https://gitpod.slack.com/archives/C04HBF1GWJ0/p1673335543513649)

## Related Issue(s)

Fixes #

## How to test

- Open workspace in preview env
- Exec `window.gitpod.loggedUserID` in browser DevTool Console to see it's assigned

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
